cloudify_cli/config/parser_config.py

```diff
@@ -166,6 +178,111 @@ def parser_config():
                     }
                 }
             },
+            'snapshots': {
+                'help': "Manages Cloudify's Snapshots",
+                'sub_commands': {
+                    'create': {
+                        'arguments': {
+                            '-s,--snapshot-id': argument_utils.remove_completer(
+                                snapshot_id_argument(
+                                    hlp='A unique id that will be assigned to the created snapshot'
+                                )
+                            ),
+                            '--include-metrics': {
+                                'dest': 'include_metrics',
+                                'action': 'store_true',
+                                'default': False,
+                                'help': 'Specify this flag to include metrics data'
```

Comment:
What do you mean 'Exclude metrics from snapshots.". Metrics are excluded by default.